### ISSUE TITLE
Re-schedule Monthly Meeting with Dr. Crowe

### ISSUE THREAD

**Issue by jamesmrollins** [lzim/teampsd#1088]
**1. Add description:** 
Need to re-establish monthly meeting between Dr. Zimmerman and Dr. Crowe.  It fell off the calendar last month for some reason.  
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by staceypark** [comment]
@jamesmrollins @lijenn This was plated over to Jenn. I think she had a hard time reaching Chris?

**Comment by jamesmrollins** [comment]
@staceypark @lijenn .  I would recommend an introduction email so he knows who Jenn is, and then a quick follow up by Jenn tomorrow morning, as it is top of mind.  Also, he is EST, so is done by 2 PM our time.

**Comment by lijenn** [comment]
@jamesmrollins @staceypark Yeah, I sent him an introduction email along with the request to select potential dates and times back in the 2nd week of February. He seemed really busy as he would respond to a few of my instant message greetings, but then never got back to me with a date/time.

Just saw the email you sent, James. I will send Dr. Crowe a follow up with potential date/times by tomorrow morning.
